CDP4Common/AutoGenHelpers/DefaultPermissionProvider.cs

```diff
@@ -55,7 +55,7 @@ public class DefaultPermissionProvider : IDefaultPermissionProvider
             { ClassKind.ArchitectureElement, ParticipantAccessRightKind.SAME_AS_SUPERCLASS },
             { ClassKind.ArrayParameterType, ParticipantAccessRightKind.SAME_AS_SUPERCLASS },
             { ClassKind.Attachment, ParticipantAccessRightKind.SAME_AS_CONTAINER },
-            { ClassKind.Behavior, ParticipantAccessRightKind.SAME_AS_SUPERCLASS },
+            { ClassKind.Behavior, ParticipantAccessRightKind.SAME_AS_CONTAINER },

```

Comment:
Hmm Ive already landed this on the GCD branch. Not sure why it is being diffed here.

https://github.com/RHEAGROUP/COMET-SDK-Community-Edition/commit/17df2bfe33a617fe3f4bbd3bfc515a4a150a59e3